addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2193,8 +2193,9 @@ options.registry.Box = options.Class.extend({
      */
     _computeWidgetState(methodName, params) {
         if (methodName === 'setShadow') {
+            const shadowClass = this.$target.hasClass('shadow');
             const shadowValue = this.$target.css('box-shadow');
-            if (!shadowValue || shadowValue === 'none') {
+            if (!shadowClass || !shadowValue || shadowValue === 'none') {
```

Comment:
@nboulif Your analysis in the commit message seems right but it proves this is the wrong fix: if the card in the jobs page have a default shadow, that shadow should be shown by the editor panel and thus this code here seems right. It should not rely on the precense of a class but on the actual box-shadow being used on the clicked element. So if there is something to fix, it should be around the setShadow method to make it possible to actually remove the shadow of the job page.

Reply:
@qsm-odoo Thanks for the review! I updated the fix :+1: